apps/site/src/components/user-avatar.tsx

```diff
@@ -16,16 +20,19 @@ export const UserAvatar: FunctionComponent<UserAvatarProps> = ({ user }) => {
       alignItems="center"
       justifyContent="center"
       sx={{
-        width: 32,
-        height: 32,
+        minWidth: size,
+        minHeight: size,
         borderRadius: "50%",
         backgroundColor: ({ palette }) => palette.gray[20],
         borderWidth: 1,
         borderStyle: "solid",
         borderColor: ({ palette }) => palette.gray[30],
       }}
     >
-      <Box component="span" sx={{ color: ({ palette }) => palette.gray[70] }}>
+      <Box
+        component="span"
+        sx={{ color: ({ palette }) => palette.gray[70], fontSize: size / 2 }}
```

Comment:
With the merge of  #912, this span now has a line-height equal to the height of the element. Can you change that to `size` like you did for the width and height, please?

Reply:
Yes, I've addressed this as part of [this merge commit](https://github.com/blockprotocol/blockprotocol/pull/901/commits/eb81571ac712ed5413179edf32c8f51cb6016b93)